osidb/exception_handlers.py

```diff
@@ -39,4 +44,7 @@ def exception_handler(exc, context):
             }
             return Response(data, status=status.HTTP_409_CONFLICT)
 
+    if isinstance(exc, DjangoValidationError):
+        exc = DRFValidationError(as_serializer_error(exc))
```

Comment:
Just a question, but why is there no logger?

Reply:
as this just pre-process exception and passes it down the `drf_exception_handler` at the end of the exception_handler function, this handler then handles the exception accordingly, normal API errors are handled with the response to the user and are usually not worth logging I guess and anything unknown and special will be reraised as internal server error 500 which raises exception anyway.

Reply:
We can theoretically log the exception everytime, but this might result in some duplicities, not sure whether to do this or not

Reply:
Up to you :)

Reply:
We can add it later to check the results on stage (and keep production clean for now).